src/Phoenix/PhoenixClient.ts

```diff
@@ -71,28 +81,53 @@ export class PhoenixClient {
   private async getContainerStatusAsync(containerData: IContainerData): Promise<ContainerInfo> {
     try {
       const runContainerStatusAsync = async () => {
+        TelemetryProcessor.traceStart(Action.PhoenixHeartBeat, {
+          dataExplorerArea: Areas.Notebook,
+        });
         const response = await window.fetch(
-          `${this.getPhoenixContainerPoolingEndPoint()}/${containerData.dbAccountName}/${containerData.forwardingId}`,
+          `${this.getPhoenixControlPlanePathPrefix()}/${containerData.forwardingId}`,
           {
             method: "GET",
             headers: PhoenixClient.getHeaders(),
           }
         );
         if (response.status === HttpStatusCodes.OK) {
+          TelemetryProcessor.traceSuccess(Action.PhoenixHeartBeat, {
+            dataExplorerArea: Areas.Notebook,
+          });
           const containerStatus = await response.json();
           return {
             durationLeftInMinutes: containerStatus?.durationLeftInMinutes,
             notebookServerInfo: containerStatus?.notebookServerInfo,
             status: ContainerStatusType.Active,
           };
         } else if (response.status === HttpStatusCodes.NotFound) {
+          const error = "Container got disconnected from the workspace";
+          Logger.logError(error, "PhoenixClient/getContainerStatusAsync");
```

Comment:
>Logger.logError [](http://example.com/codeflow?start=10&length=15)

do we need this?
guess elemetry processor will send the error to appinsights as well whhich is what logerror does?

Reply:
This is the logger that gets displayed in portal console

Reply:
i dont think we want that.. console logs should be only for development